content/blog/2022-03-05-newsletter.de.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Der XMPP Newsletter Februar 2022
+author: Anoxinon e.V.
+date: 2022-03-05
+categories: ['Newsletter']
+---
+
+_Diese Ausgabe wurde von [Anoxinon e.V.](https://anoxinon.de/) aus der XMPP Community übersetzt._
+
+Willkommen zum XMPP-Newsletter für den Februar 2022!
+
+Wir hoffen, dass du bisher einen guten Übergang in das neue Jahr hattest und dich auf die neue Ausgabe freust! Wir denken, dass diese Ausgabe über die Neujahrsfeiertage einen gewissen Umfang angenommen hat.
```

Comment:
Irgendwas passt hier nicht. Das steht doch nicht so im Original?

Reply:
yep, you are right. That was from the newsletter before.
I made some changes.